CharlieBackend.Api/Controllers/CoursesController.cs

```diff
@@ -42,14 +42,18 @@ public async Task<ActionResult<CreateCourseDto>> PostCourse(CreateCourseDto cour
         }
 
         /// <summary>
-        /// Gets all cources
+        /// Gets courses
         /// </summary>
+        /// <param name="isActive">
+        /// 1. If IsActive true method return all active Courses.
+        /// 2. If IsActive false method return all not active Courses.
+        /// 3. If IsActive null method return all courses. </param>
```

Comment:
Minor grammar corrections
```suggestion
        /// 1. If IsActive is true – method returns all active Courses.
        /// 2. If IsActive is false – method returns all not active Courses.
        /// 3. If IsActive is null – method returns all courses. </param>
```